src/test/kotlin/net/maizegenetics/phgv2/cli/InitdbTest.kt

```diff
@@ -17,7 +17,7 @@ class InitdbTest {
         }
 
         // Comment these out if you need to look at the logs files created by ProcessBuilder
```

Comment:
Comment out what exactly?

Reply:
Comment out the teardown code.  It removes the folders where the files were written.